myning/chapters/enter_time_machine.py

```diff
@@ -62,11 +62,13 @@ def play():
             # Janky, but this will exit to the run.sh loop which will reboot the game. Basically purges the memory of the game.
             exit(123)
 
-        if option == "View Current Macguffin":
-            pick(
-                ["Cool cool cool"],
-                f"XP boost: {player.macguffin.exp_percentage} Mineral value boost: {player.macguffin.store_percentage}",
-            )
+        if option == "View Potential Macguffin":
+            value = get_total_value()
+            xp_boost = round(get_boost(player.macguffin.exp_boost, value) * 100, 2)
+            mineral_boost = round(get_boost(player.macguffin.mineral_boost, value) * 100, 2)
```

Comment:
these calculations could be done outside of the loop and shared with the other option